packages/node/src/lib/novu.ts

```diff
@@ -39,37 +58,139 @@ export class Novu extends EventEmitter {
       },
     });
 
-    this.subscribers = new Subscribers(this.http);
-    this.environments = new Environments(this.http);
-    this.events = new Events(this.http);
-    this.trigger = this.events.trigger;
-    this.bulkTrigger = this.events.bulkTrigger;
-    this.broadcast = this.events.broadcast;
-    this.changes = new Changes(this.http);
-    this.layouts = new Layouts(this.http);
-    this.notificationGroups = new NotificationGroups(this.http);
-    this.notificationTemplates = new NotificationTemplates(this.http);
-    this.feeds = new Feeds(this.http);
-    this.topics = new Topics(this.http);
-    this.integrations = new Integrations(this.http);
-    this.messages = new Messages(this.http);
+    this.subscribers = new Subscribers(this);
```

Comment:
i am not sure i am a bit fan of passing 'this' to all of the properties only to use the http methods. this way we will pass this ref to all of the `Novu` properties with no actual reason unless i am missing something. 

potentially we could add the http methods and handleError to the WithHttp class.

anyways would love to hear other's opinions on the matter.

Reply:
@djabarovgeorge 
I reverted back to WithHttp in `feeds`, `subscribers` and `changes` and added axios custom request getRequest, postRequest etc 

What do you think about this approach?

Reply:
any reason we can't use WithHttp in all of the classes?

Reply:
Not a specific reason, We can go ahead in other classes as well. if you like this approach

Reply:
Removed error handling as per Pawel's suggestion
Reverted back these changes